src/main/java/seedu/address/model/person/Avatar.java

```diff
@@ -22,8 +22,8 @@
 public class Avatar {
 
     public static final String MESSAGE_AVATAR_CONSTRAINTS =
-            "Please specify the filepath for the avatar image eg. C:\\image.png\n (for Windows), "
-            + "/User/username/path/to/image.jpg (for MacOS). "
+            "Please specify the filepath for the avatar image eg. i/C:\\image.png\n (for Windows), "
```

Comment:
Might need to change the prefix from `i/` to `av/`

Reply:
requested changes for this part!